### ISSUE TITLE
Service stuck at activating

### ISSUE THREAD

**Issue by hajkenek** [spocon/lovspotify#22]
I have problem with 1.6.3 version. Service is not starting. Logs below:

```
systemd[1]: Started lovspotify.
java[6014]: no main manifest attribute, in /opt/lovspotify/lovspotify-1.6.3.jar
systemd[1]: lovspotify.service: Main process exited, code=exited, status=1/FAILURE
systemd[1]: lovspotify.service: Failed with result 'exit-code'.
systemd[1]: lovspotify.service: Scheduled restart job, restart counter is at 2.
systemd[1]: Stopped lovspotify.
```

Version 1.6.2 is running, but stuck at connecting to speaker.

File 'manifest' in v1.6.3 is empty unlike in v1.6.2. Idk it is related, but I noticed that.

**Comment by keyboardsmoke** [comment]
I am having this issue too, no idea how to downgrade to 1.6.2

**Comment by BigTA78** [comment]
Yep, same issue here too. I can confirm that 1.6.2 appears to be active. It's installed on a fresh Pi with the instructions on [this](https://spocon.github.io/spocon/) page. I can see the Connect device in my Spotify app and connect to it. It appears to play music (I can't check as my DAC is not yet set up and I don't have speakers connected. The web GUI doesn't work. http://localhost:8080 gets a connection refused. Under the API section of config.toml, you can set the port to 8080,  but this gives a 404 error.
